src/shell/js/composeui-fdc3/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+"compilerOptions": {
+    "moduleResolution": "node",
```

Comment:
including "types": [ "node", "jest" ], here should resolve the current issue on the CI regarding types:
see: https://github.com/morganstanley/ComposeUI/blob/main/src/shell/js/composeui-node-launcher/tsconfig.json